src/coreComponents/physicsSolvers/wavePropagation/ElasticWaveEquationSEMKernel.hpp

```diff
@@ -315,12 +315,13 @@ struct DampingMatrixKernel
         if( facesDomainBoundaryIndicator[f] == 1 && freeSurfaceFaceIndicator[f] != 1 )
         {
           constexpr localIndex numNodesPerFace = FE_TYPE::numNodesPerFace;
-          real64 xLocal[ numNodesPerFace ][ 3 ];
-          for( localIndex a = 0; a < numNodesPerFace; ++a )
+          real64 xLocal[ 4 ][ 3 ];
```

Comment:
What don't you change `numNodesPerFace` line 329 then?

Reply:
It is needed later when we compute the actual damping term. The values need to be computed for all d.o.f. (including higher-order nodes), but with this PR, only the actual mesh nodes are needed. Hence the change. But to make it more understandable, I have moved the definition of `numNodesPerFace` closer to where it is actually used